src/applications/static-pages/sass/modules/_m-education-wizard.scss

```diff
@@ -49,7 +48,30 @@
 }
 
 .wizard-fieldset {
-  input[type="checkbox"], input[type="radio"] {
+  input[type='checkbox'],
+  input[type='radio'] {
     margin-left: -4rem;
   }
 }
+
+.wizard-edith-nourse-content {
+  @media (min-width: $medium-screen) {
+    margin-right: 64px;
+  }
+}
```

Comment:
The design system happens to include [responsive prefixes](https://design.va.gov/utilities/margins#responsive-prefixes), so you could use:

```html
<ul class="medium-screen:vads-u-margin-right--8" >
```

Reply:
For a very visual example I tried using:
`<ul className="medium-screen:vads-u-color--orange">`

Which should change the color of the text of the `ul` to orange when the screen is at least `768px`, however this does not occur at any page size, unlike on other pages where I have used the responsive prefixes in the past.